.ci/scripts/doca_install.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
IMO 
Script readability is compromised by attempt to create unified execution flow

Reply:
Agree. It can be more readable and clear. 
@dpressle, please let me know if you agree to merge this script as is, and I will refactor it in a separate PR.

Reply:
yes agree